generators/form/templates/formSimple.js.ejs

```diff
@@ -0,0 +1,50 @@
+<%- include('partials/formHeader', {formNumber: formNumber}); %>
+
+const formFields = {
```

Comment:
We might want to chat about this. But I really like this approach of declaring all the form fields up to and referring to them in the `formConfig`. Repeating the same string all over the place gives me the heebie-jeebies.

Reply:
I like this idea. Would this just contain the form fields from the schema and not `view:fields`?≈

Reply:
I think that also including the view-only fields is smart. Maybe prepend `view:fields` with `view`? So something like:
```js
const formFields = {
  viewPetName: 'view:petName'
}
```